pkg/controllers/disruption/helpers.go

```diff
@@ -88,7 +88,7 @@ func SimulateScheduling(ctx context.Context, kubeClient client.Client, cluster *
 		return client.ObjectKeyFromObject(p), nil
 	})
 
-	results := scheduler.Solve(logging.WithLogger(ctx, operatorlogging.NopLogger), pods).TruncateInstanceTypes(pscheduling.MaxInstanceTypes)
+	results := scheduler.Solve(controllerruntime.LoggerInto(ctx, logr.New(logr.Discard().GetSink())), pods).TruncateInstanceTypes(pscheduling.MaxInstanceTypes)
```

Comment:
Can this be a static logger so that we don't have to re-generate the logger every time we want to throw out the logs? Consider assigning this to what's currently stored in the static `operatorlogging.NopLogger` value